package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ably/ui",
-  "version": "14.1.3",
+  "version": "14.1.5",
```

Comment:
I approved this but forgot - this will need to be `14.1.4` as `.5` does not yet exist

Reply:
`14.1.4` now exists though so this can go in once the version has been updated

Reply:
That's why I waited on #411 first :)